application/utils.py

```diff
@@ -78,3 +78,12 @@ def send_message(msg_content, user_email, html=None):
               "html":html,
               "subject": "Validation Service update",
               "text": msg_content})
+
+def do_try(fn, default=None):
```

Comment:
I think you can remove this change now

Reply:
Done

Reply:
I cannot comment on your line above. 

> I think we should have default values for every field of the object in case something goes wrong/an information is not retrieved.

This is already the case.

- The default value of validity is **invalid**

 https://github.com/Ghesselink/ifc-pipeline-validation/blob/19deaf9e934d8e6e79445b9bf86657cf68aeba32/application/bsdd_utils.py#L61